### ISSUE TITLE
Luigi 2.7.5 no longer works on Windows with python-daemon version > 2.1.2

### ISSUE THREAD

**Issue by pybokeh** [spotify/luigi#2525]
With Luigi's version 2.75 and its dependency on python-daemon version > 2.1.2, Luigi can no longer be installed on Windows machine.  After downgrading python-daemon to 2.1.2, all is good and expected.

Traceback:
```
Collecting python-daemon<3.0 (from luigi==2.7.5)
  Using cached https://files.pythonhosted.org/packages/99/2a/75fe6aa7086e838570f29899f674e7896a42be26d9fff33f90d990e599d2/python-daemon-2.2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user123\AppData\Local\Temp\pip-install-2ae_dkbk\python-daemon\setup.py", line 34, in <module>
        level=0, fromlist=main_module_fromlist)
      File "C:\Users\user123\AppData\Local\Temp\pip-install-2ae_dkbk\python-daemon\daemon\__init__.py", line 38, in <module>
        from .daemon import DaemonContext
      File "C:\Users\user123\AppData\Local\Temp\pip-install-2ae_dkbk\python-daemon\daemon\daemon.py", line 19, in <module>
        import pwd
    ModuleNotFoundError: No module named 'pwd'
```

**Comment by dlstadther** [comment]
Is this not a duplicate of #2494 ?

**Comment by pybokeh** [comment]
@dlstadther I believe it is not.  If I understand correctly, that issue is Linux specific with docutils failing to install.  My/this issue is Windows specific where the python daemon is running in a process that is only valid for a UNIX/POSIX operating system.  It is trying to import pwd module.  pwd is only present on Unix systems, not on Windows systems.  I am not a developer or software engineer, so I could be wrong.

**Comment by pybokeh** [comment]
https://docs.python.org/3/library/pwd.html  -  seems to indicate pwd module is UNIX only.

**Comment by dlstadther** [comment]
Oh, right. Apologizes. I see now that what i saw was your post of the same stacktrace on that issue thread.

I understand the different issues.

Unfortunately, I don't have the time capacity right now to dive into Luigi bugs. However, I'm happy to review possible fixes.

Thanks for raising!

**Comment by honnix** [comment]
Will do a release very soon to push out #2536. If possible please help verify and close the issue. Thanks! @pybokeh

**Comment by honnix** [comment]
Released 2.7.9.
